pkg/services/sqlstore/migrations/dashboard_public_mig.go

```diff
@@ -65,4 +65,13 @@ func addPublicDashboardMigration(mg *Migrator) {
 
 	// rename table
 	addTableRenameMigration(mg, "dashboard_public_config", "dashboard_public", "v2")
+	// some migrations (like AddColumn) need the table object to be passed in. So we need to make sure its updated.
+	dashboardPublicCfgV2.Name = "dashboard_public"
+
+	mg.AddMigration("add enable_annotations column", NewAddColumnMigration(dashboardPublicCfgV2, &Column{
+		Name:     "enable_annotations",
```

Comment:
I think @juanicabanas is right, lets make this `annotations_enabled`

Reply:
naming is hard

Reply:
Hardest thing in computer science 😂 !

Reply:
always makes me think of the quote `There are 2 hard problems in computer science: cache invalidation, naming things, and off-by-1 errors.`